docs/node/run-your-node/paratime-node.mdx

```diff
@@ -330,6 +292,129 @@ accessible on the internet (for `TCP` and `UDP` traffic).
 
 :::
 
+#### Custom Bundle Registry
+
+If you want to download bundles using a registry other than the
+[Oasis Bundle Registry], add the URL of your desired registry to the runtime
+configuration.
+
+```
+# ... sections not relevant are omitted ...
+runtime:
+    runtimes:
+        - id: {{ runtime_identifier }}
+        # Custom registries for all ParaTimes
+        registries:
+            - {{ url_to_registry }}
+```
+
+Before using the configuration, replace the following variables:
+
+* `{{ runtime_identifier }}`: You can find the runtime identifier in the Network
+Parameters page ([Mainnet], [Testnet]).
+* `{{ url_to_registry }}`: Url to your custom registry. The registry must ensure
+that all metadata files are accessible through a bundle registry URL, as
+metadata URLs are formed by appending the metadata file name, i.e. the bundle
+checksum, to this URL. Therefore, the bundle registry URL doesn't need to be
+valid endpoint, only the constructed metadata URLs need to be valid.
+For more information, see [ADR25].
+
+[ADR25]: ../../adrs/0025-bundle_hot_loading/
+[Oasis Bundle Registry]: https://github.com/oasisprotocol/bundle-registry
+
+### Manual Bundle Installation
```

Comment:
The "Manual Bundle Installation" is common for both ParaTime node and ParaTime client node chapters right? In this case I'd avoid redundancy and put this section in the ParaTime node chapter only somewhere at the end (Advanced?). And then reference it from the client node chapter.